src/libs/ajax.js

```diff
@@ -1166,19 +1172,6 @@ const GoogleBilling = signal => ({
     const response = await fetchGoogleBilling(`${billingAccountName}/projects`, _.merge(authOpts(), { signal }))
     const json = await response.json()
     return _.map('projectId', json.projectBillingInfo)
-  },
-  getBillingInfo: async project => {
-    const response = await fetchGoogleBilling(`projects/${project}/billingInfo`, _.merge(authOpts(), { signal }))
-    return response.json()
-  },
-  changeBillingAccount: async ({ projectId, newAccountName }) => {
-    const name = `projects/${projectId}/billingInfo`
-    const response = await fetchGoogleBilling(name,
-      _.mergeAll([
-        authOpts(), { signal, method: 'PUT' },
-        jsonBody({ billingEnabled: true, billingAccountName: newAccountName, name, projectId })
-      ]))
-    return response.json()
```

Comment:
why are we removing these?

Reply:
They are unused, and, more importantly, we actively don't want to use these. We would much rather use the `.Billing` endpoints

Reply:
The reason the one above is staying is that we don't currently have endpoints to check billing accounts in Terra. We would need that capability before removing the listBillingAccounts endpoint from here.

Reply:
aha, I see `Billing.changeBillingAccount` now.
> The reason the one above is staying is that we don't currently have endpoints to check billing accounts in Terra. We would need that capability before removing the listBillingAccounts endpoint from here.

should we have a ticket for this?

Reply:
I don't know if it is something we want to handle? @gpolumbo-broad is talking to product about what we want to do to support billing account managers.